views/randomBeer.hbs

```diff
@@ -0,0 +1,8 @@
+<div>
+    {{#each beersFromApi}}
+        <img src="{{this.image_url}}" alt="randomBeer">
```

Comment:
Like on the `beers` page, you are receiving from the API a string with the image URL, you shouldn't use double quotes to get the image.